utils-intervalrdd/src/main/scala/org/bdgenomics/utils/interval/rdd/IntervalPartition.scala

```diff
@@ -103,7 +103,7 @@ protected class IntervalPartition[K <: Interval[K], V: ClassTag](protected val a
 private[rdd] object IntervalPartition {
 
   def apply[K <: Interval[K], K2 <: Interval[K2], V: ClassTag](iter: Iterable[(K, V)]): IntervalPartition[K, V] = {
-    val array = iter.toArray
+    val array = iter.toArray.sortBy(_._1)
```

Comment:
I would rather make the change suggested in [my comment on the issue](https://github.com/bigdatagenomics/utils/issues/99#issuecomment-271410275). Let me know if you'd rather do it the way you've done here, and we can sort it out.